GOVERNANCE.md

```diff
@@ -4,51 +4,170 @@ This document defines governance policies for the Kiali project.
 
 ## Maintainers
 
-Kiali Maintainers have write access to the Kiali GitHub repositories https://github.com/kiali.
-They can merge their own patches or patches from others. The current maintainers can be found in https://github.com/orgs/kiali/teams/maintainers.
+Kiali Maintainers have write access to the [Kiali GitHub repositories](https://github.com/kiali).
+They can merge their own patches or patches from others. Maintainers collectively manage the project's
+resources and contributors. Maintainers are elegible to become admins of specific repositories, or owners
+of the Kiali organization.
 
-This privilege is granted with an expectation of responsibility: maintainers care about the Kiali project and want to help it grow and improve. Above the ability to contribute changes, a maintainer has demonstrated the ability to collaborate well with the team, assign appropriate code reviewers, contribute high-quality code, and be thorough and timely with fixes, tests and documentation.
+These privileges are granted with an expectation of responsibility: maintainers care about the Kiali project and want to help it grow and improve. Above the ability to contribute changes, a maintainer has demonstrated the ability to collaborate well with the team, assign appropriate code reviewers, contribute high-quality code, and be thorough and timely with fixes, tests and documentation.
 
 A maintainer is a contributor to the Kiali project's success and a citizen helping the project succeed.
 
-## Becoming a Maintainer
+Current list of maintainers (alphabetically ordered):
 
-To become a maintainer you need to demonstrate the following:
+* [aljesusg](https://github.com/aljesusg) - _Owner_
```

Comment:
I put a general comment about "owners/admins".
I see them more like a detail depending of the task you may need, but I wouldn't add it on the file.